res/sections/25-Lez07032018.tex

```diff
@@ -17,134 +24,175 @@ \subsection{Analisi di mercato}
-che le start-up che entrano siano di successo, ovvero si cerca di ottenere un
-\textbf{track record}\footnote{Storico delle \textbf{exit} di successo delle
-start-up di un incubatore} alto.
-
-Gli \textbf{FFF}\footnote{Ovvero i \textit{friendly, friends and fools},
-indicando rispettivamente i parenti/persone strette, gli amici e gli ``stolti''
-ovvero chi si fa convincere d'investire subito nell'azienda.} possono aiutare
-nella fase di bootstrap.\todo{Non ha detto nient'altro a riguardo!}
-
- \item a questo punto è possibile cominciare ad andare da investori seri per
-poter finire sul mercato. Si parte con attività di visibilità e marketing per
-mettere in risalto ciò che viene venduto. I \textbf{Business Angels} possono
-aiutare: essi sono un primo gruppo di investori, a cui poi seguono, nella fase
-successiva, i Venture Capital. Per le differenze tra i due
-vedesi~\ref{startup:crearestartup:bavsvc}. Quando entrano in gioco i business
-angel si dice che le quote dei founders si \textit{diluiscono}: il valore della
-start-up aumenta, anche se le quote di partecipazione dei fondatori
-diminuiscono. Questo, verso la fine, potrebbe portare al termine delle quote di
-maggioranza dal punto di vista di founders, anche se possono continuare a
-essere considerate azionisti di maggioranza.
+ nasce cioè la start-up. Fin prima, infatti, l'idea era solo su carta.
+ L'obiettivo è la creazione del \textbf{MVP}, ovvero il \textit{Minimum Viable
+ Product}, cioè un prodotto non completo con delle funzionalità chiave minime (
+ non è un prototipo, in quanto un prototipo è il primo prodotto completo).
+ L'MVP presenta caratteristiche minime in quanto permette la sua produzione in
+ tempi ristretti: un errore di molta start-up è voler uscire con un prodotto
+ troppo rifinito per essere un MVP, e ciò richiede troppo tempo per essere
+ fatto.
+ 
+ Lo scopo di un MVP è di ottenere feedback dal mercato e se ci sono clienti
+ interessati. Ciò è fondamentale in quanto approcciandosi ad aree nuove di
+ mercato è importante ottenere delle giuste direzione di mercato. Nel caso in
+ cui il mercato indichi che la direzione di mercato scelta è errata è
+ necessario fare \textbf{pivoting}, ovvero cambiare la direzione in cui ci si
+ stava muovendo. Nel mondo delle start-up è normale che ci siano dei continui
+ cambi di direzione: centrare le necessità del business model al primo colpo è
+ praticamente impossibile. Spesso se una startup non fa pivoting vuol dire che
+ da poca importanza ai feedback del mercato oppure non ne vengono proprio
+ raccolti.
+
+ Ci sono vari aiuti ad una startup per il bootstrap.
+
+ I \textbf{co-working} sono dei luoghi, strutture e ambienti che messi a
+ disposizione ad aziende/start-up in cambio di qualcosa.
+ Questi spazi sono importanti perché permettono ai fondatori di start-up di
+ avere degli ambienti di lavoro in cui è possibile sviluppare e lavorare, molto
+ spesso entrare in contatto con altre start-up dove idee potrebbero venire
```

Comment:
io metterei `in cui è possibile sviluppare, lavorare e, molto spesso, entrare in contatto con`. Però vedi te quale ti suona meglio come frase!